logging/cluster-logging-release-notes.adoc

```diff
@@ -11,8 +11,41 @@ toc::[]
 .Logging Compatibility
 The {logging-title} is provided as an installable component, with a distinct release cycle from the core {product-title}. The link:https://access.redhat.com/support/policy/updates/openshift#logging[Red Hat OpenShift Container Platform Life Cycle Policy] outlines release compatibility.
 
+[id="cluster-logging-release-notes-5-5-1"]
+== Logging 5.5.1
+This release includes link:https://access.redhat.com/errata/RHSA-2022:6344[OpenShift Logging Bug Fix Release 5.5.1].
+
+[id="openshift-logging-5-5-1-enhancements"]
+=== Enhancements
+* This enhancement adds an Aggregated Logs tab to the Pod Details page of the OpenShift WebConsole when the RedHat OpenShift Logging Console Plugin is in use. This enhancement is only available on {product-title} 4.10 and later. (link:https://issues.redhat.com/browse/LOG-2647[LOG-2647])
```

Comment:
```suggestion
* This enhancement adds an *Aggregated Logs* tab to the *Pod Details* page of the {product-title} web console when the RedHat OpenShift Logging Console Plugin is in use. This enhancement is only available on {product-title} 4.10 and later. (link:https://issues.redhat.com/browse/LOG-2647[LOG-2647])
```
- Please correct the formatting and product name attribute issues here.
- Is `RedHat OpenShift Logging Console Plugin` the correct name for the product? I've never seen `RedHat` as one word. Also, should `OpenShift` here be a `{product-title}` attribute? Please confirm.